MIT.txt

```diff
@@ -0,0 +1,17 @@
+Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
This file probably ought to be somewhere else as well because it isn't MIT right now.

Reply:
Where do you think we should put it? My main intention is to provide a copy of the license text. Do you think it would be fine to just include it as part of the licensing document?